kubeflow/core/jupyterhub.libsonnet

```diff
@@ -220,6 +157,19 @@ c.RemoteUserAuthenticator.header_name = 'x-goog-authenticated-user-email'",
                     name: "REPO_NAME",
                     value: repoName,
                   },
+                  {
+                    name: "KF_AUTHENTICATOR",
+                    value: jupyterHubAuthenticator,
+                  },
+                  {
+                    name: "KF_PVC_LIST",
+                    value: disks,
+                  },
+                  if cloud == "gke" then
+                    {
+                      name: "GCP_SECRET_NAME",
```

Comment:
Why does JupyterHub need a GCP secret to add PVs to pods?

Reply:
Is this to add the GCP secret to the JupyterPods?

Reply:
This secret is mounted to all jupyter notebook pods. It is not used by jupyterhub.